Improve viewport unit readability

jtomwilson opened the pull request:
Ran into a readability issue when trying to complete challenge. Instead of using `width: 80vw` was attempting `vw: 80vw` based on the initial layout. I believe either this change or the example that was added but doesn't seem to be pushed yet will help.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

jtomwilson committed:
Improve viewport unit readability

Ran into a readability issue when trying to complete challenge. Instead of using `width: 80vw` was attempting `vw: 80vw` based on the initial layout. I believe either this change or the example that was added but doesn't seem to be pushed yet will help.

xyozio commented on curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-typography-responsive.english.md:
```diff
@@ -10,7 +10,7 @@ forumTopicId: 301141
 Instead of using <code>em</code> or <code>px</code> to size text, you can use viewport units for responsive typography. Viewport units, like percentages, are relative units, but they are based off different items. Viewport units are relative to the viewport dimensions (width or height) of a device, and percentages are relative to the size of the parent container element.
 The four different viewport units are:
-<ul><li><code>vw: 10vw</code> would be 10% of the viewport's width.</li><li><code>vh: 3vh</code> would be 3% of the viewport's height.</li><li><code>vmin: 70vmin</code> would be 70% of the viewport's smaller dimension (height vs. width).</li><li><code>vmax: 100vmax</code> would be 100% of the viewport's bigger dimension (height vs. width).</li></ul>
+  <ul><li><code>width: 10vw</code> would be 10% of the viewport's width.</li><li><code>height: 3vh</code> would be 3% of the viewport's height.</li><li><code>width: 70vmin</code> or <code>height: 70vmin</code>would be 70% of the viewport's smaller dimension (height vs. width).</li><li><code>width:100vmax</code> or <code>height:100vmax</code> would be 100% of the viewport's bigger dimension (heigh ...
```
Please remove indentetaion before `<ul><li><code>width...`

jtwilson0294 replied:
Okay I think I fixed the indent - I'm rather new to the contributing front so let me know if I did something incorrectly!

jtomwilson committed:
Fix indent on line 13

Fix accidental indent added during initial pull

thecodingaviator commented on curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-typography-responsive.english.md:
```diff
@@ -10,7 +10,7 @@ forumTopicId: 301141
 Instead of using <code>em</code> or <code>px</code> to size text, you can use viewport units for responsive typography. Viewport units, like percentages, are relative units, but they are based off different items. Viewport units are relative to the viewport dimensions (width or height) of a device, and percentages are relative to the size of the parent container element.
 The four different viewport units are:
-<ul><li><code>vw: 10vw</code> would be 10% of the viewport's width.</li><li><code>vh: 3vh</code> would be 3% of the viewport's height.</li><li><code>vmin: 70vmin</code> would be 70% of the viewport's smaller dimension (height vs. width).</li><li><code>vmax: 100vmax</code> would be 100% of the viewport's bigger dimension (height vs. width).</li></ul>
+<ul><li><code>width: 10vw</code> would be 10% of the viewport's width.</li><li><code>height: 3vh</code> would be 3% of the viewport's height.</li><li><code>width: 70vmin</code> or <code>height: 70vmin</code>would be 70% of the viewport's smaller dimension (height vs. width).</li><li><code>width:100vmax</code> or <code>height:100vmax</code> would be 100% of the viewport's bigger dimension (height  ...
```
Not only can these units be used only on `width` and `height` but also work with `font-size`

raisedadead reviewed (commented):
@thecodingaviator can we apply the suggestions and land this?

thecodingaviator commented:
@ojeytonwilliams @RandellDawson What do you think about this one? On second look, the file looks good to me in its original form without any changes.

raisedadead commented:
Did we have any open issues for this?

RandellDawson commented:
@raisedadead I do not think these changes improves the challenge at all.  The only improvement I might suggest would be to remove the colon between unit and the example.  The way it is written on `master` makes it seem as if the whole thing is actual code to be used.

jtwilson0294 commented:
@RandellDawson That was my goal with the pull request. A friend and I were working through the course and both got stumped trying to use 'vw:' instead of 'width: 1vw', etc. Thought it may be beneficial to tweak to prevent anyone else getting tripped up.

moT01 reviewed (changes requested):
I, personally, don't think there's a problem with how it currently is. If you read through it, there's an example that should make it pretty clear:
```
Here is an example that sets a body tag to 30% of the viewport's width.
body { width: 30vw; }
```

What about something like this... ?
    `vw`: `10vw` would be 10% of the viewport's width.
    `vh`: `3vh` would be 3% of the viewport's height.
    `vmin`: `70vmin` would be 70% of the viewport's smaller dimension (height vs. width).
    `vmax`: `100vmax` would be 100% of the viewport's bigger dimension (height vs. width).
separate the highlights so it doesn't look like a single block of code - or something along those lines.

Or maybe just remove the first part like this...
    `10vw` would be 10% of the viewport's width.
    `3vh` would be 3% of the viewport's height.
    `70vmin` would be 70% of the viewport's smaller dimension (height vs. width).
    `100vmax` would be 100% of the viewport's bigger dimension (height vs. width).

To be clear, I don't like the changes on this PR - I think these changes might actually make it more confusing.

Edit: how about this...
`vw` (viewport width): `10vw` would be 10% of the viewport's width.
`vh` (viewport height): `3vh` would be 3% of the viewport's height.
`vmin` (viewport minimum): `70vmin` would be 70% of the viewport's smaller dimension (width or height).
`vmax` (viewport maximum): `100vmax` would be 100% of the viewport's larger dimension (width or height).

jtwilson0294 commented:
I like the example and your suggestions. When we were going through this particular lesson I don't believe there was an example yet, and I think the example more than helps clear up what tripped us up earlier. Thanks @moT01.

moT01 commented:
So do you want to do something with this @jtwilson0294? - I like my first or third example. Anyone else want to weigh in? I think there would be some benefit to separating the code highlighting.

jtomwilson committed:
Improve viewport unit readability

Modified original update to viewport unit challenge thanks to feedback from other contributors.

jtomwilson committed:
Fixed spacing

jtwilson0294 commented:
Okay so I ~believe~ I updated this. I used your last example @moT01. When I committed the changes to my own branch they showed up here in the PR. Is there anything else I need to do or was it all automatic? Thanks for the help and discussion everyone.

RandellDawson commented:
@moT01 If you are fine with the latest changes, can you approve?

moT01 reviewed (approved):
LGTM 🎉 Thanks for fixing this up @jtwilson0294 🎉

ojeytonwilliams commented:
@jtwilson0294  Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:
